tests/component_djangorestframework/test_application.py

```diff
@@ -12,190 +12,169 @@
+    url_module_path = "django.core.urlresolvers"
+    process_request_method = ".process_request"
+    process_view_method = ".process_view"
+    process_response_method = ".process_response"
 
 if DJANGO_VERSION >= (2, 0):
-    url_resolver_cls = 'URLResolver'
+    url_resolver_cls = "URLResolver"
 else:
-    url_resolver_cls = 'RegexURLResolver'
+    url_resolver_cls = "RegexURLResolver"
 
 _scoped_metrics = [
-        ('Function/django.core.handlers.wsgi:WSGIHandler.__call__', 1),
-        ('Python/WSGI/Application', 1),
-        ('Python/WSGI/Response', 1),
-        ('Python/WSGI/Finalize', 1),
-        (('Function/django.middleware.common:'
-                'CommonMiddleware' + process_request_method), 1),
-        (('Function/django.contrib.sessions.middleware:'
-                'SessionMiddleware' + process_request_method), 1),
-        (('Function/django.contrib.auth.middleware:'
-                'AuthenticationMiddleware' + process_request_method), 1),
-        (('Function/django.contrib.messages.middleware:'
-                'MessageMiddleware' + process_request_method), 1),
-        (('Function/%s:' % url_module_path +
-                '%s.resolve' % url_resolver_cls), 1),
-        (('Function/django.middleware.csrf:'
-                'CsrfViewMiddleware' + process_view_method), 1),
-        (('Function/django.contrib.messages.middleware:'
-                'MessageMiddleware' + process_response_method), 1),
-        (('Function/django.middleware.csrf:'
-                'CsrfViewMiddleware' + process_response_method), 1),
-        (('Function/django.contrib.sessions.middleware:'
-                'SessionMiddleware' + process_response_method), 1),
-        (('Function/django.middleware.common:'
-                'CommonMiddleware' + process_response_method), 1),
+    ("Function/django.core.handlers.wsgi:WSGIHandler.__call__", 1),
+    ("Python/WSGI/Application", 1),
+    ("Python/WSGI/Response", 1),
+    ("Python/WSGI/Finalize", 1),
+    (("Function/django.middleware.common:" "CommonMiddleware" + process_request_method), 1),
+    (("Function/django.contrib.sessions.middleware:" "SessionMiddleware" + process_request_method), 1),
+    (("Function/django.contrib.auth.middleware:" "AuthenticationMiddleware" + process_request_method), 1),
+    (("Function/django.contrib.messages.middleware:" "MessageMiddleware" + process_request_method), 1),
+    (("Function/%s:" % url_module_path + "%s.resolve" % url_resolver_cls), 1),
+    (("Function/django.middleware.csrf:" "CsrfViewMiddleware" + process_view_method), 1),
+    (("Function/django.contrib.messages.middleware:" "MessageMiddleware" + process_response_method), 1),
+    (("Function/django.middleware.csrf:" "CsrfViewMiddleware" + process_response_method), 1),
+    (("Function/django.contrib.sessions.middleware:" "SessionMiddleware" + process_response_method), 1),
+    (("Function/django.middleware.common:" "CommonMiddleware" + process_response_method), 1),
```

Comment:
Could we fix the concatenated strings?  (I realize this is the end result of the linter doing its thing)

```
    (("Function/django.middleware.common:CommonMiddleware" + process_request_method), 1),
    (("Function/django.contrib.sessions.middleware:SessionMiddleware" + process_request_method), 1),
    (("Function/django.contrib.auth.middleware:AuthenticationMiddleware" + process_request_method), 1),
    (("Function/django.contrib.messages.middleware:MessageMiddleware" + process_request_method), 1),
    (("Function/%s:" % url_module_path + "%s.resolve" % url_resolver_cls), 1),
    (("Function/django.middleware.csrf:CsrfViewMiddleware" + process_view_method), 1),
    (("Function/django.contrib.messages.middleware:MessageMiddleware" + process_response_method), 1),
    (("Function/django.middleware.csrf:CsrfViewMiddleware" + process_response_method), 1),
    (("Function/django.contrib.sessions.middleware:SessionMiddleware" + process_response_method), 1),
    (("Function/django.middleware.common:CommonMiddleware" + process_response_method), 1),
```